castle/migrations/0029_challenge.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I don't like it because there are 200 lines in this pull request alone that have a completely-zero effect on the logic. 

I leave it to the admin to decide if it is appropriate to have these in source control. My core point is that they are not source, and so should not be, but I would accept a ruling on the grounds of necessity.